Core/Systems/AltUseSystem.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FunnyExperience.Core.Systems.TreeSystem;
+using System.Linq;
+using Terraria.Audio;
+using Terraria.ModLoader.IO;
+
+namespace FunnyExperience.Core.Systems
+{
+	public class AltUseSystem : ModPlayer
```

Comment:
Changing item m1/m2 inputs to simply be listeners for skills would also let es eliminate this and use the built-in cooldown for skills